custom_components/plugwise_usb/models.py

```diff
@@ -0,0 +1,196 @@
+"""Models for the Plugwise USB integration."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntityDescription,
+)
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.components.switch import SwitchDeviceClass, SwitchEntityDescription
+from homeassistant.const import (
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    EntityCategory,
+    UnitOfEnergy,
+    UnitOfPower,
+    UnitOfTime,
+)
+from homeassistant.helpers.entity import EntityDescription
+
+from .const import STICK, USB_MOTION_ID, USB_RELAY_ID
+
+
+@dataclass
+class PlugwiseRequiredKeysMixin:
+    """Mixin for required keys."""
+
+    plugwise_api: str
```

Comment:
This part is no longer needed after the split.

Also below all `plugwise_api=STICK,` lines can be removed.

Reply:
Tracking via #3